modules/dxp/apps/portal-workflow-kaleo-forms/portal-workflow-kaleo-forms-web/src/main/java/com/liferay/portal/workflow/kaleo/forms/web/internal/portlet/KaleoFormsAdminPortlet.java

```diff
@@ -267,6 +268,10 @@ private void _renderKaleoProcess(RenderRequest renderRequest)
 				WorkflowTaskManagerUtil.getWorkflowTask(
 					themeDisplay.getCompanyId(), workflowTaskId));
 		}
+
+		renderRequest.setAttribute(
```

Comment:
@JanisZhang I think you can add a method `KaleoFormsAdminDisplayContext.getWorkflowDefinitionManager` so that you don't need to add it to the request.

Reply:
@dantewang force-pushed. thanks!